assets/app/scripts/directives/alerts.js

```diff
@@ -5,7 +5,9 @@ angular.module('openshiftConsole')
     return {
       restrict: 'E',
       scope: {
-        alerts: '='
+        alerts: '=',
+        // If truthy, hide the close button.
+        persistent: '=?'
```

Comment:
I'm fine with the change, although Patternfly calls them "inline" and "toast" notifications.